policyengine_us/variables/gov/states/pa/dhs/tanf/eligibility/pa_tanf_resources_eligible.py

```diff
@@ -0,0 +1,17 @@
+from policyengine_us.model_api import *
+
+
+class pa_tanf_resources_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "PA TANF resources eligible"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.PA
+
+    def formula(spm_unit, period, parameters):
+        resource_limit = parameters(
+            period
+        ).gov.states.pa.dhs.tanf.resource_limit
+        total_assets = spm_unit("pa_tanf_countable_resources", period)
+        return total_assets <= resource_limit
```

Comment:
If making a new Python variable, though, please use `resources` for consistency.
```suggestion
        return spm_unit("pa_tanf_countable_resources", period) <= resource_limit
```